index.html

```diff
@@ -1,298 +1,348 @@
-    </head>
-    <body >
-        <header id="header-container">
-            <div class="logo-container"><a href="#">Welcome</a></div>
-
-            <nav>
-                
-                    <img class="Mobile-menu" src="../setup-and-mobile-version-skeleton/images/Union.png" alt="">
-                
-               <!-- <li>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Setup and Mobile Version Skeleton</title>
+    <link rel="stylesheet" href="../setup-and-mobile-version-skeleton/style.css">
+</head>
+
+<body>
+    <header id="header-container">
+        <div class="logo-container"><a href="#">Welcome</a></div>
+
+        <nav>
+
+            <img class="Mobile-menu" src="../setup-and-mobile-version-skeleton/images/Union.png" alt="">
+
+            <!-- <li>
                     <a>Home</a>
                     <a>Portfolio</a>
                     <a>Contact</a>
                 </li>-->
-            </nav>
+        </nav>
 
-        </header>
-        <div id="Header-background">
+    </header>
+    <div id="Header-background">
 
-            <section class="headline">
+        <section class="headline">
 
-              <div class="intro-container">
-                    <div class="headline-intro">
-                        <h1>I’m Massaillon
-                            Glad to see you!</h1>
-                    </div>
-                    <div class="intro-desc-sumary">
-                        <p>I’m a software developer! I can help you build a product , 
-                        feature or website Look through some of my work and experience! 
+            <div class="intro-container">
+                <div class="headline-intro">
+                    <h1>I’m Massaillon
+                        Glad to see you!</h1>
+                </div>
+                <div class="intro-desc-sumary">
+                    <p>I’m a software developer! I can help you build a product ,
+                        feature or website Look through some of my work and experience!
                         If you like what you see and have a project you need coded,
-                         don’t hestiate to contact me.  </p>
-                    </div>
-                
+                        don’t hestiate to contact me. </p>
+                </div>
+
 
                 <div class="social-area">
                     <h4 class="social-network">LET’S CONNECT</h4>
                     <div class="social-icon">
                         <ul>
                             <li>
-                                <a href="#" class="twitter"><img src="../setup-and-mobile-version-skeleton/images/Twiter.svg" alt=""></a>
+                                <a href="#" class="twitter"><img
+                                        src="../setup-and-mobile-version-skeleton/images/Twiter.svg" alt=""></a>
                             </li>
```

Comment:
Why not use [fontawesome](https://fontawesome.com/) for these icons rather than images. [Fontawesome](https://fontawesome.com/) are font and load quite easier because they ar lighter than images.